src/kubernetes.cc

```diff
@@ -140,9 +137,11 @@ MetadataUpdater::ResourceMetadata KubernetesReader::GetNodeMetadata(
   return MetadataUpdater::ResourceMetadata(
       std::vector<std::string>{k8s_node_name},
       k8s_node,
+      /*full_resource_name=*/"",
 #ifdef ENABLE_KUBERNETES_METADATA
-      MetadataStore::Metadata(config_.MetadataIngestionRawContentVersion(),
```

Comment:
Should we remove this config parameter now?

Reply:
Removed